### DISCUSSION TITLE
Puan Toplamanın Farklı Yolları

### DISCUSSION THREAD

**Discussion by Eemreecan** [IBAX-io/go-ibax#688] [Rewards]
Puan toplamak için rt, star ve fork dışında yollar nelerdir acaba. 20k puanım var 30k civarında puanı olanlar ekstra olarak ne yaptılar acaba. Bu tartışma altına yardımlaşalım.

**Comment by Trajedi032** [comment]
Kıskançlar onlara yetismeyelim diye

**Comment by abuzercan** [comment]
bilgisi olan paylaşırsa seviniriz

**Comment by Omertastan2036** [comment]
Başka yok en yüksek puanlı kişi 35 k zaten o da Twitter çekilişleri ile olmustur
